source/common/filter/http/filter_config_discovery_impl.cc

```diff
@@ -106,7 +117,7 @@ void FilterConfigSubscription::onConfigUpdate(
     throw EnvoyException(fmt::format(
         "Unexpected number of resources in ExtensionConfigDS response: {}", resources.size()));
   }
-  const auto& filter_config = dynamic_cast<const envoy::config::core::v3::TypedExtensionConfig&>(
+  auto& filter_config = dynamic_cast<const envoy::config::core::v3::TypedExtensionConfig&>(
```

Comment:
why remove the const here? we cast to a const ref

Reply:
sorry, did not notice.